rules/CodingStyle/Rector/Stmt/NewlineAfterStatementRector.php

```diff
@@ -151,6 +151,10 @@ public function refactor(Node $node): ?Node
             }
         }
 
+        if ($this->nodesToRemoveCollector->isActive()) {
```

Comment:
Couldn't this be moved to the top of the refactor function to reduce a little bit of runtime?

Reply:
moved to top https://github.com/rectorphp/rector-src/pull/1350/commits/e2056ec8a4d324aa457a558788e6613d755dfeb0